src/Utils/Api.js

```diff
@@ -28,4 +28,19 @@ export const getArticleComments = (id) => {
         return data.comments
 
     })
+}
+
+export const postComment = (id, usernameInput, commentInput) => { 
+    const commentToAdd = {
+        username: usernameInput,
+        body: commentInput
+      };
+    return newsAPI.post(`/articles/${id}/comments`, {
+        username: usernameInput,
+        body: commentInput
+      }).then((response) => {
+        return response
+    }).catch((err)=> {
+        return Promise.reject((err.response.data.msg))
+    })
```

Comment:
Nice handling of error here - don't forget to connect this up to remove optimistic rendering if it fails.